cfme/tests/containers/test_smart_management.py

```diff
@@ -74,29 +82,39 @@ def wait_for_tag(obj_inst):
                                forced_streams=['5.7'],
                                unblock=lambda test_item: test_item.obj != Container)])
 def test_smart_management_add_tag(provider, test_item):
-
+    logger.info("Setting smart mgmt tag to {obj_type}".format(obj_type=test_item.obj.__name__))
     # validate no tag set to project
     if test_item.obj is ContainersProvider:
         obj_inst = provider
     else:
         obj_inst = test_item.obj.get_random_instances(provider, count=1).pop()
 
+    logger.info("Selected object is \"{obj_name}\"".format(obj_name=obj_inst.name))
+
     regex = r"([\w\s|\-|\*]+:([\w\s|\-|\*])+)|(No.*assigned)"
     try:
         # Remove all previous configured tags for given object
+        logger.info('Starting cleaning old tags from '
+                    'object "{obj_name}"'.format(obj_name=obj_inst.name))
         obj_inst.remove_tags(obj_inst.get_tags())
+        logger.info("All smart management tags was removed successfully")
     except RuntimeError:
+        logger.info("Fail to remove tags, checking if no tag set")
+
         # Validate old tags formatting
-        assert re.match(regex, wait_for_tag(obj_inst).text_value), \
-            "Tag formatting is invalid! "
+        assert re.match(regex, wait_for_tag(obj_inst).text_value), (
```

Comment:
This line continuation seems odd to me, ``\`` is acceptable here.

Reply:
On all the other reviews that i had, the reviewer ask to use `(`, do you have any coding standard that I can use? You MUST be consistent!

this comments are very needless

Reply:
I just expressed my opinion and didn't mark my comment as required, no need to call it "very needless" (I see you later edited that statement).

Generally this applies:
*The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.*

Adding parentheses just for line breaks to expressions where parentheses are not normally used is considered odd by many, feel free to disagree.
See e.g. http://legacy.python.org/dev/peps/pep-3125/
*Note that it is always possible to parenthesize an expression, but it can seem odd to parenthesize an expression that needs parentheses only for the line break:*
```python
assert val>4, (
    "val is too small")
```